### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by KostinP**
Tip: The easiest way to watch changes is by commit

### CODE REVIEW

**Review by lalagvana — APPROVED** [review]

**Review by ifelseelif — APPROVED** [review]

File: DigitalOffice/Services/CheckRightsService/Commands/GetRightsListCommand.cs [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ﻿using LT.DigitalOffice.CheckRightsService.Commands.Interfaces;
-using System.Collections.Generic;
+using LT.DigitalOffice.CheckRightsService.Models;
 using LT.DigitalOffice.CheckRightsService.Repositories.Interfaces;
 using Microsoft.AspNetCore.Mvc;
-using LT.DigitalOffice.CheckRightsService.Models;
+using System.Collections.Generic;
```
Comment (djino97) [inline DigitalOffice/Services/CheckRightsService/Commands/GetRightsListCommand.cs:5]: First connect system namespaces and than namespaces created in the our project

File: DigitalOffice/Services/ProjectService/Models/Project.cs [hunk 2]
```diff
@@ -6,7 +6,7 @@ namespace LT.DigitalOffice.ProjectService.Models
     public class Project
     {
         public string Name { get; set; }
-        public IEnumerable<Guid> ManagersIds { get; set; }
-        public IEnumerable<Guid> WorkersIds { get; set; }
+        public List<Guid> ManagersIds { get; set; }
+        public List<Guid> WorkersIds { get; set; }
     }
```
Comment (djino97) [inline DigitalOffice/Services/ProjectService/Models/Project.cs]: We removed field Managers, we added new fields for this in the db models.

File: DigitalOffice/Services/UserService/UserService.csproj [hunk 3]
```diff
@@ -16,7 +16,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <Folder Include="Models" />
     <Folder Include="Controllers\" />
   </ItemGroup>
```
Comment (djino97) [inline DigitalOffice/Services/UserService/UserService.csproj]: Why is this included folder?

File: DigitalOffice/Services/ProjectService/Models/Project.cs (hunk 2 above)
Comment (KostinP) [reply DigitalOffice/Services/ProjectService/Models/Project.cs]: Apparently they forgot to fix it.

Comment (KostinP) [reply DigitalOffice/Services/ProjectService/Models/Project.cs:11]: Do we have such a rule? Can I have a proof?

File: DigitalOffice/Services/UserService/UserService.csproj (hunk 3 above)
Comment (KostinP) [reply DigitalOffice/Services/UserService/UserService.csproj]: Fixed.

File: DigitalOffice/Services/CheckRightsService/Commands/GetRightsListCommand.cs (hunk 1 above)
Comment (KostinP) [reply DigitalOffice/Services/CheckRightsService/Commands/GetRightsListCommand.cs:5]: @djino97  said that it was more convenient for him.
I leave it like this for now because this is the default behavior (VS).

File: DigitalOffice/Services/ProjectService/Models/Project.cs (hunk 2 above)
Comment (KostinP) [reply DigitalOffice/Services/ProjectService/Models/Project.cs]: Fixed.

**Review by djino97 — APPROVED** [review]

**Review by MihailKulikov — APPROVED** [review]

**Review by seasmail — APPROVED** [review]

**Review by mekk1t — APPROVED** [review]
